```sh
$ make ci-test
/Applications/Xcode.app/Contents/Developer/usr/bin/make install
cd cmd/ecspresso && go build -ldflags "-s -w -X main.Version=v1.5.1+1-g9bc093c -X main.buildDate=2021-04-25T16:51:53+0900" -trimpath
install cmd/ecspresso/ecspresso `go env GOPATH`/bin/ecspresso
cd tests/ci && PATH=/Users/**/go/bin:PATH /Applications/Xcode.app/Contents/Developer/usr/bin/make test
ecspresso --config config.yaml --envfile envfile verify --debug
ecspresso: error: duplicate long flag --suspend-auto-scaling, try --help
make[1]: *** [verify] Error 1
make: *** [ci-test] Error 2
```

適当に作っちゃいましたが冷静に `make ci-test` が落ちていたので後で手を入れます...

--- a/cmd/ecspresso/main.go
+++ b/cmd/ecspresso/main.go
@@ -49,6 +49,7 @@ func _main() int {
 		DryRun:               scale.Flag("dry-run", "dry-run").Bool(),
 		DesiredCount:         scale.Flag("tasks", "desired count of tasks").Required().Int64(),
 		SkipTaskDefinition:   boolp(true),
+		SuspendAutoScaling:   scale.Flag("suspend-auto-scaling", "set suspend to auto-scaling attached with the ECS service").IsSetByUser(&isSetSuspendAutoScaling).Bool(),
 		ForceNewDeployment:   boolp(false),
 		NoWait:               scale.Flag("no-wait", "exit ecspresso immediately after just deployed without waiting for service stable").Bool(),
 		UpdateService:        boolp(false),
@@ -217,6 +218,9 @@ func _main() int {
 	case "refresh":
 		err = app.Deploy(refreshOption)
 	case "scale":
+		if !isSetSuspendAutoScaling {
+			scaleOption.SuspendAutoScaling = nil
+		}
 		err = app.Deploy(scaleOption)
 	case "status":
 		err = app.Status(statusOption)

